plugins/woocommerce-admin/client/task-lists/components/terms-of-service.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { Text } from '@woocommerce/experimental';
+import interpolateComponents from '@automattic/interpolate-components';
+import { __, sprintf } from '@wordpress/i18n';
+import { Link } from '@woocommerce/components';
+
+export const TermsOfService = ( { buttonText } ) => (
```

Comment:
Extracted to a new component because the TOS are always the same except for the label of the button that should be clicked.